apps/i18n/common/en_us.json

```diff
@@ -470,6 +470,7 @@
   "congratsCertificateThanks": "Thanks for submitting!",
   "congratsCertificateContinue": "Now, keep going with our other courses, or see more options to Learn beyond an Hour below.",
   "congratsNextLevelHeading": "Graduate to the next level",
+  "congratsProjectSelected": "Congratulations! This project has been selected to be a featured project or an exemplar project!",
```

Comment:
nit: "featured project or an exemplar project" would "featured as an exemplar project" be clearer? I'm just thinking of whether someone reading it might think it's one or the other.  Just a thought, and totally fine with being overruled.

Reply:
Actually, they are two different things. A curriculum writer can freeze projects that will be used as exemplars. I think this is why we first implemented the 'frozen' state.